docs/getting_started/02_breakdown.md

```diff
@@ -35,25 +35,36 @@ _Verifier.toml_ contains public in/output values computed when executing the Noi
 
 _Nargo.toml_ contains the environmental options of your project. It contains a "package" section and a "dependencies" section.
 
-#### Package section
-
-The package section requires a number of fields including:
-
-- name - the name of the package
-- type - can be "bin", "lib", or "contract" to specify whether its a binary, library or Aztec contract
-- authors
-- compiler_version - specifies the version of the compiler to use. This is not currently enforced by the compiler, but will be in future versions.
-
-For example:
+Example Nargo.toml:
 
-```toml
+```yaml
```

Comment:
I'm confused why the `yaml` syntax here?

Reply:
(also shows in other places)

Reply:
```suggestion
```toml
```

Reply:
It was `toml` in a number of places, but the syntax highlighting doesn't support `toml` so i switched it to `yaml`, since there are some similarities. Wdyt, should we keep `yaml` to get some highlighting or just label it `toml` and get none?